rai/client.go

```diff
@@ -318,6 +325,11 @@ func unmarshal(rsp *http.Response, result interface{}) error {
 		}
 
 		return errors.Errorf("unhandled unmarshal type %T", result)
+	case generated.MetadataInfo:
+		dstValues := reflect.ValueOf(result).Elem()
```

Comment:
Is `dstValues` not already assigned in line 281? I'm not sure about Go scoping, but I guess it doesn't matter here as we are mutating the `result` argument...

Reply:
yep it's already assigned, this line is redundant, cleaning it up :+1: